README.md

```diff
@@ -29,22 +29,27 @@ If you're interested in becoming a maintainer of this collection, refer to the [
 
 ## Communication
 
-We announce important development changes and releases through Ansible's [The Bullhorn newsletter](https://github.com/ansible/community/wiki/News#the-bullhorn). If you are a contributor, be sure you are [subscribed](https://eepurl.com/gZmiEP).
+> `GitHub Discussions` feature is disabled in this repository. Use the `network` tag on the forum in the [Project Discussions](https://forum.ansible.com/new-topic?title=topic%20title&body=topic%20body&category=project&tags=network) or [Get Help](https://forum.ansible.com/new-topic?title=topic%20title&body=topic%20body&category=help&tags=network) category instead.
```

Comment:
Why even mention GitHub discussions if it's disabled?

Reply:
I think it's better to use just the network tag link, not the new topic link. SEO will pick up common links and raise visibility of that content on search results.

Reply:
I don't remember the original rationale, took if from another repo. Yes, let's remove it.